docs/development_guide/README.md

```diff
@@ -75,6 +75,48 @@ Run `go get -d ./...` to download all required Go packages.
 
 Also the project requires Docker. See how to [install Docker](https://docs.docker.com/engine/install/).
 
+## Getting started
```

Comment:
What if this section came after the Code Formatting section? The instructions for setting up the env file and the review of the makefile would then come before using all of the commands/steps together. The section could be called "Putting it all together" or "Get deploying your OSM changes" (the section could event fall in the testing section). Just a thought!

Reply:
definitely a could have a better name here :-).  

I personally like to get right in when I am contributing a new repo but if I added a TOC at the top that would make it easy to find and jump to

Reply:
updated!